### PR TITLE
Update README.md for pfSense

### PR DESCRIPTION
**Pull request by swordfish6975**
Just installed it this is what I had to do.

### CODE REVIEW

File: packaging/installer/README.md [hunk 1]
```diff
@@ -352,12 +352,23 @@ Note first three packages are downloaded from the pfSense repository for maintai
...
 pkg install bash
 pkg install e2fsprogs-libuuid
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/python36-3.6.8_2.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.13.0.txz
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/python36-3.6.9.txz
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/libuv-1.31.0.txz
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.15.0.txz
 ```
 
+Edit
```
Comment (joelhans) [inline packaging/installer/README.md]: Let's not force users to use a specific text editor. This section could be simplified with the following:

```
Edit `/usr/local/etc/netdata/netdata.conf` and change `bind to = 127.0.0.1` to `bind to = 0.0.0.0`.
```

File: packaging/installer/README.md [hunk 2]
```diff
@@ -352,12 +352,23 @@ Note first three packages are downloaded from the pfSense repository for maintai
...
+```
+vi /usr/local/etc/netdata/netdata.conf
+```
+change "bind to = 127.0.0.1" to "bind to = 0.0.0.0"
+
 To start Netdata manually run `service netdata onestart`
 
+Check its working:
```
Comment (joelhans) [inline packaging/installer/README.md]:
```suggestion
Visit the Netdata dashboard to confirm it's working:
```

**Comment by Fohdeesha** [comment]
I submitted a PR with these changes a little cleaner ( for instance linking to the raw libuv package url is not necessary, it's in the default pfsense repo and a raw link will only be one more URL to 404 when freebsd updates the repo) - https://github.com/netdata/netdata/pull/6768

**Comment by cakrit** [comment]
After resolving the conflicts, I saw that the information is already in the installation README, so closing this one. Thank you @swordfish6975 !

**Pull request closed without merging**